src/index.ts

```diff
@@ -19,6 +20,8 @@ export * from '@/composables';
 
 export { contextColors, type ContextColorsType } from '@/consts/colors';
 
+export { translationKeys } from '@/i18n/keys';
```

Comment:
This hopefully can help us exclude the keys from the unused ones in `eslint.config.ts` without having to manually track them